lib/spack/spack/ci.py

```diff
@@ -1734,7 +1731,7 @@ def setup_spack_repro_version(repro_dir, checkout_commit, merge_commit=None):
             )
 
             if git.returncode != 0:
-                tty.error("Unable to merge {0}".format(merge_commit))
+                tty.error(f"Unable to merge {merge_commit}")
```

Comment:
(no change needed) Note: every change above here is a refactoring of `x.format()` to `f"x"`, minus the global definition of `IS_WINDOWS`.